docs/content/comparisons/druid-vs-sql-on-hadoop.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: doc_page
+---
+
+Druid vs SQL-on-Hadoop (Hive/Impala/Drill/Spark SQL/Presto)
+===========================================================
```

Comment:
This section would benefit from being more specific about which techs support which things, when a particular feature is not universal. (instead of saying things like "Most" and "Many" engines)